.circleci/config.yml

```diff
@@ -15,6 +20,12 @@ jobs:
     - run:
         command: bazel test --curses=no --color=yes --show_timestamps //...
         name: Bazel tests
+    - save_cache:
+        paths:
+        - ~/.cache/bazel
```

Comment:
I'm not sure if this is the correct path to use, or if we should store _more_ that I may be missing.

Reply:
You can take a look at CircleCI to see if that's doing what you want.

I'm currently notijcing that it's not processing any test results: https://app.circleci.com/pipelines/github/apple/rules_pkl/153/workflows/095e1f1a-3697-47a9-b47c-eef445940ad8

Reply:
It's not processing any tests because it's not getting past the `save_cache` step.